File: lib/limelight/builtin/players/text_area.rb
#- Copyright © 2008-2010 8th Light, Inc. All Rights Reserved.
#- Limelight and all included source files are distributed under terms of the GNU LGPL.

on_cast do
  text_area = Limelight::UI::Model::Inputs::TextAreaPanel.new
  panel.add(text_area)
  @text_area = text_area
end

attr_accessor :text_area #:nodoc:

File: src/limelight/builtin/players/TextArea.java
package limelight.builtin.players;

import limelight.ui.events.panel.PanelEvent;
import limelight.ui.model.PropPanel;
import limelight.ui.model.inputs.TextAreaPanel;

public class TextArea
{
  public void install(PanelEvent event)
  {
    TextAreaPanel input = new TextAreaPanel();
    final PropPanel panel = (PropPanel)event.getRecipient();
    panel.add(input);
  }
}
